auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/EditOrganisationsController.kt

```diff
@@ -27,27 +25,32 @@ class EditOrganisationsController(
         route.post { updateUserOrganisations(call) }
     }
 
-    // This endpoint takes a list of organisation codes which should contain all and only the organisations of which
-    // the user should be a member after the execution of the endpoint
+    // This endpoint takes a list of organisation codes which should contain all and only the domain organisations of which
+    // the user should be a member after the execution of the endpoint. Note that if the user is a member of any organisations
+    // not in their domain, these will NOT be affected by this endpoint.
     private suspend fun updateUserOrganisations(call: ApplicationCall) {
         val session = call.principal<OAuthSession>()!!
         val callingUser = userLookupService.lookupUserByCn(session.userCn)
         logger.atInfo().log("Updating organisations for user {}", session.userCn)
 
         val requestedOrganisations = call.receive<DeltaUserOrganisations>().userSelectedOrgs
```

Comment:
Maybe worth renaming `userSelectedOrgs` in the request body too, `selectedDomainOrganisationCodes`?